### PR TITLE
Updating the curation interface update to reorder level

### PR DESCRIPTION
**Pull request by olaaustine**
[G2P-342](https://www.ebi.ac.uk/panda/jira/browse/G2P-342)

### CODE REVIEW

File: src/utility/CurationConstants.js [hunk 1]
```diff
@@ -212,3 +212,12 @@ export const MechanismSynopsisAttribs = [
 ];
 
 export const MechanismSupportAttribs = ["inferred", "evidence"];
+
+export const ConfidenceAttribs = [
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/utility/CurationConstants.js:216]: suggesting this to make it more clear

File: src/components/curation/Confidence.vue [hunk 2]
```diff
@@ -7,6 +8,13 @@ export default {
     inputPublications: Object,
   },
   emits: ["update:justification", "update:level"],
+  methods: {
+    reOrderConfidenceCategory() {
+      return this.attributesData.confidence_category.sort(
+        (a, b) => ConfidenceAttribs.indexOf(a) - ConfidenceAttribs.indexOf(b)
+      );
+    },
+  },
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/components/curation/Confidence.vue:17]: I feel this can be a computed property

**Review by dglemos — COMMENTED** [review]
The changes look good!

File: src/views/update/UpdateConfidence.vue [hunk 3]
```diff
@@ -82,6 +83,12 @@ export default {
           console.log(error);
         });
     },
+    reOrderConfidenceCategoryList() {
+      return this.attributesData.confidence_category.sort(
+        (a, b) =>
+          ConfidenceAttribsOrder.indexOf(a) - ConfidenceAttribsOrder.indexOf(b)
+      );
+    },
```
Comment (seeta-ramaraju) [inline src/views/update/UpdateConfidence.vue:91]: please make similar changes done for Confidence.vue component

Comment (seeta-ramaraju) [inline src/views/update/UpdateConfidence.vue:91]:
```suggestion
  },
  computed: {
    reorderedConfidenceCategoryList() {
      return this.attributesData.confidence_category.sort(
        (a, b) =>
          ConfidenceAttribsOrder.indexOf(a) - ConfidenceAttribsOrder.indexOf(b)
      );
    },
```

**Review by seeta-ramaraju — APPROVED** [review]
LGTM

**Pull request merged**